### DISCUSSION TITLE
Sprite Cursor For Loop Increment.

### DISCUSSION THREAD

**Discussion by AnRich26** [Bodmer/TFT_eSPI#3462] [General]
Good day Bodmer, i have been using the lib for a while now for a bunch of projects. Recently i took on a personal project whear esp32 will be working almost overtime, however i havent even started with my project's coding and already i ran into problem. I have striped out a part i need help with however that function will be used alot in code at varias points. So i figured let me start at basic, the following code:

```
for (int i = 0; i < 127; i++) {
  Wire.beginTransmission(i);
  if (Wire.endTransmission() == 0) {
    Spr.print(i);
  }
  Spr.pushSprite(0, 0);
}
```

Basically sprite print should increment on the y-axis every 17 pixels and display something like:

```
I2C Adress:
Found 20
      40
      41
      00
      72
      76
```

I have tryd using same formula as draw arc but info is displayd ontop one another, any help will be apreciated. Thank you and blessed day.

**Comment by AnRich26** [comment]
Well got it working somewhat but still not correct using same example as above with code as follows:

```
for (int i = 0; i < 127; i++) {
  Wire.beginTransmission(i);
  if (Wire.endTransmission() == 0) {
    for (int j = 0; j < 6; j++) {
      Spr.setCursor(36, 28 + (j * 17), 2)
      Spr.print(i);
  }
}
Spr.pushSprite(0, 0);
```

On tft it prints out but same info on every line overlaping with next sensor id.
